src/main/java/org/logstash/dissect/fields/AppendField.java

```diff
@@ -0,0 +1,52 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public final class AppendField extends AbstractField {
+    private static final int ORD = 2;
+
+    private AppendField(String s, int ord) {
+        super(s, ord);
+    }
+
+    public static Field create(String s) {
+        if (hasOrdinal(s)) {
+            String[] parts = s.split("/");
+            return new AppendField(parts[0], ORD + Integer.valueOf(parts[1]));
```

Comment:
Ordinal is an integer that is used in when the saveable fields are sorted.  Its a mechanism to say some Field types sort before others but in the Append field's case the user may specify the ordinal to change the Append field sort order within the AppendField group.